ghost/core/core/server/data/migrations/versions/5.25/2022-11-24-10-37-add-email-spam-complaint-events-table.js

```diff
@@ -0,0 +1,9 @@
+const {addTable} = require('../../utils');
+
+module.exports = addTable('email_spam_complaint_events', {
+    id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+    member_id: {type: 'string', maxlength: 24, nullable: false, references: 'members.id'},
+    email_id: {type: 'string', maxlength: 24, nullable: false, references: 'emails.id'},
+    email_address: {type: 'string', maxlength: 191, nullable: false, unique: true},
```

Comment:
@allouis Should this be unique? What if an email gets multiple spam complaint events?

(ie. email sent, reported as spam, added to suppression list + event added, removed from suppression list, email sent, reported as spam, event added, 💥)

Reply:
Nice catch my man

Reply:
Could add a 'onConflict/ignore' thing to prevent that

Reply:
have removed the constraint